euphonic/spectra.py

```diff
@@ -644,6 +661,40 @@ def from_castep_phonon_dos(cls: SC, filename: str) -> SC:
             y_data*ureg('dimensionless'),
             metadata=metadata)
 
+    def broaden(self, x_width: Quantity, shape: str = 'gauss') -> SC:
+        """
+        Individually broaden each line in y_data, returning a new
+        Spectrum1DCollection
+
+        Parameters
+        ----------
+        x_width
+            Scalar float Quantity. The broadening FWHM
+        shape
+            One of {'gauss', 'lorentz'}. The broadening shape
+
+        Returns
+        -------
+        broadened_spectrum
+            A new Spectrum1DCollection object with broadened y_data
+
+        Raises
+        ------
+        ValueError
+            If shape is not one of the allowed strings
+        """
+        y_broadened = np.zeros(self._y_data.shape)
```

Comment:
A nice numpy shortcut for this (which also gets the dtype right) is `np.zeros_like(self._y_data)`. (Doesn't make much difference here, happy to leave as-is if you prefer.)

Reply:
Done!